src/osx/cocoa/utils_base.mm

```diff
@@ -191,3 +191,95 @@ wxString wxGetOsDescription()
     return true;
 }
 #endif // wxUSE_DATETIME
+
+bool wxCocoaLaunch(const char* const* argv, pid_t &pid)
+{
+    // Obtains the number of arguments for determining the size of
+    // the CFArray used to hold them
+    NSUInteger cfiCount = 0;
+    for (const char* const* argvcopy = argv; *argvcopy != NULL; ++argvcopy)
+    {
+        ++cfiCount;
+    }
+    
+    // If there is not a single argument then there is no application
+    // to launch
+    if(cfiCount == 0)
+    {
+        wxLogDebug(wxT("wxCocoaLaunch No file to launch!"));
+        return false ;
+    }
+    
+    // Path to bundle
+    wxString path = *argv++;
+    NSError *error = nil;
+    NSURL *url = [NSURL fileURLWithPath:wxCFStringRef(path).AsNSString() isDirectory:YES];
+    
+    // Check the URL validity
+    if( url == nil )
+    {
+        wxLogDebug(wxT("wxCocoaLaunch Can't open path: %s"), path.c_str());
+        return false ;
+    }
+    
+    // Loop through command line arguments to the bundle,
+    // turn them into CFURLs and then put them in cfaFiles
+    // For use to launch services call
+    NSMutableArray *params = [[NSMutableArray alloc] init];
+    for( ; *argv != NULL; ++argv )
+    {
+        // Check for '<' as this will ring true for
+        // CFURLCreateWithString but is generally not considered
+        // typical on mac but is usually passed here from wxExecute
+        if (wxStrcmp(*argv, wxT("<")) == 0)
+            continue;
```

Comment:
This doesn't make any sense to me, why should we silently ignore `<`? Of course, it shouldn't be here in the first place (this is `wxExecute()`, not `wxShell()`), but this entire check should be just removed.

Reply:
these lines and their comment comes from Ryans version from way back when, so ok, let's remove that

Reply:
@vadz ,
I guess the idea is that OSX bundle can't take input from the command line.

I will see if removing those will produce the same result.

Reply:
ok, I'm waiting on this, if it's fine without, I'll commit the changes